physics/CONV/SAMF/samfdeepcnv.f

```diff
@@ -100,16 +101,17 @@ subroutine samfdeepcnv_run (im,km,first_time_step,restart,        &
      &   prslp(:,:),  garea(:), hpbl(:), dot(:,:), phil(:,:)
       real(kind=kind_phys), dimension(:), intent(in) :: fscav
       logical, intent(in)  :: first_time_step,restart,hwrf_samfdeep,    &
-     &     progsigma,do_mynnedmf
+     &     progsigma,progomega,do_mynnedmf
       real(kind=kind_phys), intent(in) :: nthresh,betadcu,betamcu,      &
      &                                    betascu
       real(kind=kind_phys), intent(in), optional :: ca_deep(:)
       real(kind=kind_phys), intent(in), optional :: sigmain(:,:),       &
-     &     qmicro(:,:),  prevsq(:,:)
+     &     qmicro(:,:),  prevsq(:,:), omegain(:,:)
       real(kind=kind_phys), intent(in) :: tmf(:,:,:),q(:,:)
       real(kind=kind_phys), dimension (:), intent(in), optional :: maxMF
       real(kind=kind_phys), intent(out) :: rainevap(:)
-      real(kind=kind_phys), intent(out), optional :: sigmaout(:,:)
+      real(kind=kind_phys), intent(out), optional :: sigmaout(:,:),     &
```

Comment:
sigmaout and omegaout are intent(out) but are not always give a value if present, which is a CCPP rule. The fact that they're optional maybe makes that rule cloudy, though. I'm thinking that it is probably best to add `if present(sigmaout) sigmaout = 0.0_kind_phys` (same for omegaout). @dustinswales You're more involved with optional arguments, do you think this is necessary?